ptbacc2.php

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>Giải phương trình bậc hai</title>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    </head>
+    <style>
+    	input[type="text"] {
+    		margin: 10px;
+		}
+
+		input[type="submit"]:hover {
+		    cursor: pointer;
+		}
+		input[type="text"]:hover {
+		    color: red;
+		}
+    </style>
+    <body>
+        <?php 
+            $result = '';
+			if (isset($_POST['calculate'])){
+				// Bước 1: Lấy tham số
+				$a = isset($_POST['a']) ? $_POST['a'] : '';
+				$b = isset($_POST['b']) ? $_POST['b'] : '';
+				$c = isset($_POST['c']) ? $_POST['c'] : '';
+			 
+				// Bước 2: Validate và tính toán
+				$delta = ($b*$b) - (4*$a*$c);
+			 	if ($a==0 && $b==0 && $c==0){
+					$result = 'Phương trình có vô số nghiệm' ;
+				}
+				else if ($delta < 0){
```

Comment:
```suggestion
				} elseif ($delta < 0) {
```
convention như này mới chuẩn